### DISCUSSION TITLE
最近发现个好康的学习资料特地分享分享

### DISCUSSION THREAD

**Discussion by udigjbx114514** [share121/inter-knot#536] [其他]
最近在逛网站的时候，突然看到好看的学习资料，使我的脑子飞速旋转，哦天，我真的爱上它了想着独乐乐不如众乐乐，于是特地分享于绳网中，与各位绳匠欣赏一番
![img-1625266881_1723024654364](https://github.com/user-attachments/assets/ae5a9b4c-9b0c-44ab-a417-14502a2eb4c0)
https://b23.tv/9mgTqn
https://b23.tv/XI41PC
https://b23.tv/zrlMEY
https://b23.tv/Aj3Zhw

**Comment by ProxyPhaeton** [comment]
@绳网团队
快来把这个人封了！

**Comment by lllliufeng** [comment]
此番美景，邀诸位共赏

**Comment by fzdqz** [comment]
四大皆空<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >
